### PR TITLE
General css adjustments

### PR DESCRIPTION
**Pull request by bulascoskun**

### CODE REVIEW

File: src/components/BottomNavBar.tsx [hunk 1]
```diff
@@ -1,33 +1,76 @@
 import { NavLink } from 'react-router-dom';
-import discover from '../assets/nav-icon--discover.svg'
-import home from '../assets/nav-icon--home.svg'
-import chart from '../assets/nav-icon--chart.svg'
```
Comment (cris-valente) [inline src/components/BottomNavBar.tsx:4]: why?

File: src/css-modules/CategoryTiles.module.css [hunk 2]
```diff
@@ -32,23 +31,24 @@
...
 }
 
 .title {
   display: flex;
   align-items: center;
   justify-content: center;
   font-size: 10px;
-  margin-bottom: 8px;
   font-weight: 600;
-  letter-spacing: 1.3px;
```
Comment (cris-valente) [inline src/css-modules/CategoryTiles.module.css:53]: I think it looks better with letter spacing

**Review by cris-valente — CHANGES_REQUESTED** [review]

File: src/components/BottomNavBar.tsx (hunk 1 above)
Comment (bulascoskun) [reply src/components/BottomNavBar.tsx:4]: I didn't know how to use filter method img elements so I've changed them into svg's and used fill property in css to change their color

File: src/css-modules/CategoryTiles.module.css (hunk 2 above)
Comment (cris-valente) [reply src/css-modules/CategoryTiles.module.css:53]: I know but with letter spacing it looks more like the figma file. You can compare and check

File: src/components/BottomNavBar.tsx (hunk 1 above)
Comment (cris-valente) [reply src/components/BottomNavBar.tsx:4]: Would you mind on changing it back and using filter property? Converter link :
https://isotropic.co/tool/hex-color-to-css-filter/

**Pull request closed without merging**